app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/channels/DefaultChannel.kt

```diff
@@ -21,5 +22,9 @@ class DefaultChannel(
         adapter.submitList(tileData)
     }
 
+    fun removeContent(tileData: ChannelTile) {
+        // TODO
+    }
```

Comment:
This function shouldn't be needed, right?  It doesn't particularly mesh with our architecture either.  The display state after calling `removeContent` is based on previous state, whereas `setContent` handles both add and remove in a way that is stateless.

Reply:
Later commits clarify what this is doing.  I think you can ignore this comment in favor of https://github.com/mozilla-mobile/firefox-tv/pull/2235#discussion_r283029432